packages/drivers/odsp-driver-definitions/src/errors.ts

```diff
@@ -29,6 +29,11 @@ export enum OdspErrorType {
      */
     fetchTimeout = "fetchTimeout",
 
+    /*
+     * Fetch stopped by provided abort signal.
+     */
+    fetchAborted = "fetchAborted",
```

Comment:
I think you want to add this new definition and add breaking.md note that this new failure is possible, and clients need to handle it. And only after some long time we can start using it in the code.

Exception can be made if we are sure that this error is caught in our layers and never propagated to consumers. Then we can add definition and use it in the same build. It's probably true, but I'm not sure if we have good way to prove it? So maybe we should be careful and follow step above? Not sure.
Most likely it does not matter, as even if it leaks out, apps should have default handling for error types. But I'd still add breaking.md note.

Reply:
BTW, it should be easy to tell if this error propagates to consumers by searching in Kusto where this unique error message shows up